auto_nag/scripts/query_creator.py

```diff
@@ -17,16 +18,29 @@ def getTemplateValue(url):
     return parsed_template.groups()[0]
 
 
+def loadJSON(url):
+    jsonurl = urlopen(url)
+    return json.loads(jsonurl.read())
+
+
+def getVersion(jsonVersion, key):
+    # In X.Y, we just need X
+    version = jsonVersion[key].split(".")
+    return version[0]
+
+
 def getReportURL(approval_flag, span):
     a = requests.get("https://bugzilla.mozilla.org/page.cgi?id=release_tracking_report.html&q=" + approval_flag + "%3A%2B%3A" + span + "%3A0%3Aand%3A")
     return a.url
 
 
 no_nag = ";field3-1-0=status_whiteboard;type3-1-0=notsubstring;value3-1-0=[no-nag]"
-beta_version = getTemplateValue("https://wiki.mozilla.org/Template:BETA_VERSION")
-aurora_version = getTemplateValue("https://wiki.mozilla.org/Template:AURORA_VERSION")
-central_version = getTemplateValue("https://wiki.mozilla.org/Template:CENTRAL_VERSION")
-esr_version = getTemplateValue("https://wiki.mozilla.org/Template:ESR_VERSION")
+jsonContent = loadJSON("https://product-details.mozilla.org/1.0/firefox_versions.json")
+beta_version = getVersion(jsonContent, "LATEST_FIREFOX_DEVEL_VERSION")
+aurora_version = getVersion(jsonContent, "FIREFOX_AURORA")
+central_version = getVersion(jsonContent, "FIREFOX_NIGHTLY")
+esr_version = getVersion(jsonContent, "FIREFOX_ESR")
```

Comment:
this reverts us from esr52 to esr45 for the next few weeks, which seems like a bad thing (45 is essentially dead for non-chemspilly things now).  can we hold off on this patch until 45 is no more, or look at ESR_NEXT, or something  else?